cli-plugins/manager/manager_test.go

```diff
@@ -92,10 +92,14 @@ func TestErrPluginNotFound(t *testing.T) {
 func TestGetPluginDirs(t *testing.T) {
 	cli := test.NewFakeCli(nil)
 
-	expected := []string{config.Path("cli-plugins")}
+	pluginDir, err := config.Path("cli-plugins")
+	assert.NilError(t, err)
+	expected := []string{pluginDir}
 	expected = append(expected, defaultSystemPluginDirs...)
 
-	assert.Equal(t, strings.Join(expected, ":"), strings.Join(getPluginDirs(cli), ":"))
+	pluginDirs1, err1 := getPluginDirs(cli)
```

Comment:
nit: you can (and should) reuse `pluginDir` and `err` here and the same for the `2`s below.

Reply:
Yes, agreed; I see this PR still needs some squashing, so if you could make that change as part of that 👍